lib/elixir/pages/anti-patterns/process-anti-patterns.md

```diff
@@ -76,7 +76,111 @@ iex> Calculator.subtract(2, 3)
 
 ## Scattered process interfaces
 
-TODO.
+#### Note
+
+Formerly known as [Agent obsession](https://github.com/lucasvegi/Elixir-Code-Smells/tree/main#agent-obsession).
```

Comment:
I have started using adding "Additional Remarks" at the bottom (after Refactoring). Perhaps we can do the same?

Reply:
> I have started using adding "Additional Remarks" at the bottom (after Refactoring). Perhaps we can do the same for this note?

Totally agree! Could you do this locally, please?